js/src/controls/Control.ts

```diff
@@ -1,13 +1,20 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
 
-//@ts-nocheck
-
-import * as widgets from '@jupyter-widgets/base';
+import { WidgetModel, WidgetView } from '@jupyter-widgets/base';
 import L from '../leaflet';
-import * as utils from '../utils';
-export class LeafletControlModel extends widgets.WidgetModel {
-  defaults() {
+import { LeafletWidgetView } from '../utils';
+
+interface LeafletControlModelOptions {
```

Comment:
We may want to export this as it will probably be useful in other places 
```suggestion
export interface LeafletControlModelOptions {
```